.github/workflows/tests.yml

```diff
@@ -19,7 +19,8 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - run: pip install --user ruff
-    - run: ruff check --output-format=github --select="E,F,PLC,PLE,UP,W,YTT" --ignore="E721,PLC1901,S101,UP031" --target-version=py38 .
+    # Excluding `/gyp` directory as it is been checked in https://github.com/nodejs/gyp-next/ already
```

Comment:
This assumption does not make me comfortable. When you vendor-in source code, it becomes your code, and you should run your own tests on it.

Reply:
Can we run two separate checks? It would be great to surface issues in `/gyp` because we do vendor it. But in the past it has created friction because a linting error in `/gyp` code would fail all of CI.

Reply:
@legendecas Can we revert this part and re-open as a new PR? It would be nice to unblock the release from CI and figure out python linting in a separate discussion.

Reply:
Thanks for the explanation @legendecas, I didn't realized CI was failing. I think this change should be landed to make CI green and a new issue/PR opened to get two different `ruff` checks running, each with the correct configuration.